crates/git_ui/src/stash_picker.rs

```diff
@@ -432,11 +475,32 @@ impl PickerDelegate for StashListDelegate {
                     .size(LabelSize::Small),
             );
 
+        let show_button = div()
+            .id("show-button")
+            .on_mouse_up(
+                MouseButton::Right,
+                cx.listener(move |picker, _: &MouseUpEvent, window, cx| {
+                    cx.stop_propagation();
+                    picker.delegate.show_stash_at(ix, window, cx);
+                }),
+            )
+            .child(
+                IconButton::new("show_stash", IconName::Eye)
+                    .icon_size(IconSize::Small)
+                    .tooltip(Tooltip::for_action_title("Show Stash", &ShowStashItem))
+                    .on_click(cx.listener(move |picker, _, window, cx| {
+                        cx.stop_propagation();
+                        picker.delegate.show_stash_at(ix, window, cx);
+                    })),
+            )
+            .into_any_element();
+
```

Comment:
This part of the UI feels a bit strange to interact with using the mouse--could we make it have a distinct selected/hovered color (maybe by making it a `Button` or `ButtonLike`) so that there's a visual cue when you place the mouse on it?